learningresources/models.py

```diff
@@ -228,3 +202,46 @@ class Meta:  # pylint: disable=missing-docstring
             RepoPermission.add_edit_metadata,
             RepoPermission.manage_repo_users,
         )
+
+
+def get_preview_url(resource, org=None, course_number=None, run=None):
```

Comment:
It would be mostly to keep the `models.py` cleaner, but feel free to leave it here